sycl/include/CL/sycl/backend_types.hpp

```diff
@@ -0,0 +1,19 @@
+//==-------------- backend_types.hpp - SYCL backend types ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+namespace cl {
+namespace sycl {
+
+enum class backend { host, opencl, cuda };
```

Comment:
this should replace the detail::pi:Backend enum
can be done a separate PR

Reply:
Sounds like a good idea, but I think we should get this merged ASAP and do that separately as you suggest.

Reply:
Sure